lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -286,6 +287,33 @@ class ReportUIFeatures {
     this._dom.resetTemplates();
   }
 
+  /**
+   * Jumps to respective audit on page load if hash is provided in URL
```

Comment:
nit: can you add periods to the end of these lines since there are multiple lines (we don't enforce this, just my personal preference :)